src/main/java/org/owasp/esapi/logging/java/JavaLogFactory.java

```diff
@@ -156,11 +158,12 @@ public class JavaLogFactory implements LogFactory {
      * @param logApplicationName
      * @param logServerIp
      * @param logClientInfo
+     * @param omitEventTypeInLogs
      *
      * @return LogAppender instance.
      */
-    /*package*/ static LogAppender createLogAppender(boolean logUserInfo, boolean logClientInfo, boolean logServerIp, boolean logApplicationName, String appName) {
-        return new LogPrefixAppender(logUserInfo, logClientInfo, logServerIp, logApplicationName, appName);
+    /*package*/ static LogAppender createLogAppender(boolean logUserInfo, boolean logClientInfo, boolean logServerIp, boolean logApplicationName, String appName, boolean omitEventTypeInLogs) {
+        return new LogPrefixAppender(logUserInfo, logClientInfo, logServerIp, logApplicationName, appName, omitEventTypeInLogs);
```

Comment:
May wish to change this, depending  @jeremiahjstacey's take on my comments on the `LogPrefixAppender` CTOR. But at even if this remains the same here, we can't simple remove the old CTOR as the CTOR is public and thus off limits. (Once we start using Java 9 or later, we can better control this with Java modules, but until then, no changes to signatures of public or protected methods unless it is absolutely required to address a vulnerability.)